examples/decoding/plot_haxby_anova_svm.py

```diff
@@ -46,7 +46,7 @@
 
 ### Prediction function #######################################################
 
-### Define the prediction function to be used.
+# Define the prediction function to be used.
```

Comment:
AFAIK the only constraint on comment in PEP8 is to start by sharp + whitespace. Maybe something like

```Python
# Title 1
# - Title 2
# -- Title 3
```
could do ?